doc/cargo-doc-live.md

```diff
@@ -0,0 +1,40 @@
+# cargo-doc-live
+
+[cargo-doc-live] is live server version of `cargo doc`.
```

Comment:
Could you use the 1st paragraph and video from https://github.com/srid/cargo-doc-live in here?